pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -92,6 +90,7 @@ func (r *ReconcileComplianceRemediation) Reconcile(request reconcile.Request) (r
 			return reconcile.Result{}, nil
 		}
 		// Error reading the object - requeue the request.
+		reqLogger.Error(err, "Cannot retrieve remediation", "remediation", request.Name)
```

Comment:
We already log the remediation's name in line 79, with the `log.WithValues` call.